src/audio/frame.h

```diff
@@ -91,6 +91,16 @@ class FramePos final {
         return util_isfinite(m_framePosition);
     }
 
+    // returns true if a is valid and is fairly close to target (within +/- 1 frame).
+    bool isNear(mixxx::audio::FramePos target) const {
+        VERIFY_OR_DEBUG_ASSERT(isValid()) {
+            return false;
+        }
+        return target.isValid() &&
```

Comment:
Can this isValid() also an debug assert?

Reply:
IMO the caller is responsible for testing the position. It may be possible this is called with an invalid FramePos and in that case we simply return `false`.